configure-ironic.sh

```diff
@@ -24,8 +24,11 @@ fi
 
 export HTTP_PORT=${HTTP_PORT:-"80"}
 export MARIADB_PASSWORD=${MARIADB_PASSWORD:-"change_me"}
-export NUMPROC=$(cat /proc/cpuinfo  | grep "^processor" | wc -l)
-export NUMWORKERS=$(( NUMPROC < 12 ? NUMPROC : 12 ))
+# TODO(dtantsur): remove the explicit default once we get
```

Comment:
Is there any reason for us not to just pick a number here instead of making it dynamic at all?

Reply:
How beefy are the CI nodes? I guess if they have 1-2 cores, we don't want to launch 4 API workers.

Reply:
Ah, that's fair I guess.